rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+
+As a data consumer, I want to be confident that the HCA DCP has correctly processed all data files that belong together so that I know the matrices I receive have been generated correctly.
+
+## Detailed Design
+
+### Component Notification Changes
+This RFC proposes no longer using bundle-level search-based notifications on primary bundles for initiating pipelines. Instead, we propose that a **specific trigger** is used to initiate analysis pipelines. This document lists the requirements for such notifications to be actionable by analysis and makes recommendations on overall implementation paradigm with specific examples for 10X and SS2 datasets.
+
+### Notification Requirements
+For a notification to be actionable by analysis, it must fulfill specific requirements. Specifically, the notification must include information about the bundles/data that belong to the same sequencing set and potentially need to be co-processed.
+
+We propose the term **Data Aggregation and Processing Set (DAPS)** to encompass all the different alternative representations of such sets. The only requirement imposed on such these representations is to be reproducible and resolvable to an immutable sets of data. Example representations of DAPS can be:
+
+*   A listing of all relevant bundles along with versions
+*   A search query accompanied by a timestamp (to guarantee reproducibility and immutability)
+*   Other future representations as project needs evolve (e.g. linking to a particular project bundle version)
+
+Critically, the submitted DAPS will not explicitly dictate the files that have to be co-processed in pipeline instances as this would require specialized knowledge of the pipelines by the submitter. Instead, **a DAPS indicates that aggregation of data must be performed only from the list of data provided (submission envelope), but the actual aggregation is delegated to the process of analysis initiation, subject to well defined modality-specific metadata requirements**. 
+
+The imposition of specific requirements to metadata is part of this RFC proposal and a key component of the proposed DAPS solution.
+
+This approach provides a generalized signaling mechanism suitable for all foreseeable data types while accounting for data type-specific requirements.
+
+The DAPS approach has common elements with project-level bundles that have been proposed in other contexts, but differs in the following ways:
+
+1. It does not impose any specific aggregation level on the data allowing for more flexibility and accommodating future unforeseen processing scenarios
+2. Allows specification of bundles included in a DAPS in many alternative different ways (listing, search, etc…)
+3. Does not impose the creation of a new bundle type that is tied to a specific data modality
```

Comment:
Why would a new bundle type be tied to a specific data modality - bundle type definition is based on graph shape not assay type is or there some nuance I am missing?

Reply:
this needs to be reworded in a more abstract manner.